src/main/resources/com/google/api/codegen/csharp/gapic_client.snip

```diff
@@ -967,3 +969,39 @@
 
     @end
 @end
+
+# api: StaticLangApiView, settings: StaticLangSettingsView
+@private partialGrpc(api, settings)
+    @if api.hasLongRunningOperations
+        // Partial Grpc class to enable LRO client creation
+        public static partial class {@api.grpcTypeNameOuter}
+        {
+            public partial class {@api.grpcTypeNameInner}
+            {
+                /// <summary>
+                /// Creates a new instance of <see cref="Operations.OperationsClient"/> using the same call invoker as this client.
+                /// </summary>
+                /// <returns>A new Operations client for the same target as this client.</returns>
+                public virtual Operations.OperationsClient CreateOperationsClient() => new Operations.OperationsClient(CallInvoker);
+            }
+        }
+
+
+    @end
+    @join rerouted : api.reroutedGrpcClients
+        // Partial Grpc class to enable {@rerouted.typeName} client creation
+        public static partial class {@api.grpcTypeNameOuter}
+        {
+            public partial class {@api.grpcTypeNameInner}
+            {
+                /// <summary>
+                /// Creates a new instance of <see cref="{@rerouted.typeName}"/> using the same call invoker as this client.
+                /// </summary>
+                /// <returns>A new {@rerouted.typeName} for the same target as this client.</returns>
+                public virtual {@rerouted.typeName} {@rerouted.getMethodName}() => new {@rerouted.typeName}(CallInvoker);
+            }
+        }
+
```

Comment:
These appear to be needed to get the output formatting correct (ie one blank line).
Don't ask me why :(
I had a bit of a frustrating half hour figuring this out.